packages/executable/.eslintrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@lilitkarapetyan - @foxleigh81 has taken responsibility for build & linting. Can you explain why this lint resource file has been introduced?

thanks :-)

Reply:
It looks like it's just a copy of one of my eslint files, either way, it's extending the stuff I've already established so I'm fine with this. Each package should have its own linting at the moment.